### ISSUE TITLE
ID_WEB IDs translations anywhere

### ISSUE THREAD

**Issue by NarsilAnduril** [Bouni/luxtronik#86]
Hi, 

Not an issue but a question : did someone map the sensors ID's to friendly readable names ? I have no idea what a ID_WEB_Temperatur_TEE can be, and eventually, witch ones are calculations, parameters or visibilities ?

I found those lists here : https://github.com/UweTrottmann/wp-display-android

[AlphaInnotec_value-ids.txt](https://github.com/Bouni/luxtronik/files/15406247/AlphaInnotec_value-ids.txt)

[AlphaInnotec_settings-ids.txt](https://github.com/Bouni/luxtronik/files/15406283/AlphaInnotec_settings-ids.txt)

Thanks
Diego

**Comment by Bouni** [comment]
Not really, the manufacturer of the Luxtronik controller is a german company, so many names ar abbreviations for german words.

For example `ID_Einst_BWS_akt` is most likely ID_**Einst**ellung_**B**rauch**W**a**S**ser_**aktuell** which translates roughly to `ID_setting_hotwater_current`

I've never found out how the translate settings in the webinterface and the webinterface only shows a small subset of all parameters.

**Comment by Plobli** [comment]
What language do you need? The translation into German can be found here: https://loxwiki.atlassian.net/wiki/spaces/LOX/pages/1533935933/Java+Webinterface

**Comment by Bouni** [comment]
@Plobli I made that loxwiki page long time ago from the no longer developed JAVA interface, which I reverse engineered to get the texts. But all updates since then do no longer have the Java interface and therefore all new texts are unknown.
